### ISSUE TITLE
Darwin arm64 constellation fails to run on Ventura 13.4

### ISSUE THREAD

**Issue by c3d** [edgelesssys/constellation#1915]
### Issue description

The Darwin ARM build for v2.8.0,  fails to run with a message

“constellation” can’t be opened because Apple cannot check it for malicious software.
This software needs to be updated. Contact the developer for more information.

### To reproduce

1. Following [installation instructions](https://docs.edgeless.systems/constellation/getting-started/install)
2. Run `curl -LO https://github.com/edgelesssys/constellation/releases/latest/download/constellation-darwin-arm64`
3. The signature verification step fails (see below in additional info)
4. Run the installation `sudo install ./constellation-darwin-arm64 /usr/local/bin/constellation`
5. Run `constellation`

### Environment

- md5sum 8020bd3b379454336718c6f68030cc81  constellation-darwin-arm64
- `constellation version`: not available
- `constellation-conf.yaml`: not applicable
- VM type used to run Constellation: not applicable
- `uname -a` gives `Darwin myhostname 22.5.0 Darwin Kernel Version 22.5.0: Mon Apr 24 20:52:24 PDT 2023; root:xnu-8796.121.2~5/RELEASE_ARM64_T6000 arm64`

### Expected behavior

Expected the release build to be properly codesigned

### Additional info / screenshot

Cosign step failure:

```
cosign verify-blob --key https://edgeless.systems/es.pub --signature constellation-darwin-arm64.sig constellation-darwin-arm64

Error: searching log query: [POST /api/v1/log/entries/retrieve][400] searchLogQueryBadRequest  &{Code:400 Message:verifying signature: invalid signature when validating ASN.1 encoded signature}
main.go:74: error during command execution: searching log query: [POST /api/v1/log/entries/retrieve][400] searchLogQueryBadRequest  &{Code:400 Message:verifying signature: invalid signature when validating ASN.1 encoded signature}
```

<img width="253" alt="image" src="https://github.com/edgelesssys/constellation/assets/1695924/f776e1d4-8413-4260-982c-4e762372b665">

**Comment by msanft** [comment]
Hello Christophe,

Thanks for opening this issue!

**Regarding the Apple check:**
Did you download the binary from the release assets section (i.e. through your browser)?
If you use the curl snippet from our [documentation](https://docs.edgeless.systems/constellation/getting-started/install#install-the-constellation-cli), you should be able to open the binary without problems:
```bash
curl -LO https://github.com/edgelesssys/constellation/releases/latest/download/constellation-darwin-arm64
```

Cheers,
Moritz

**Comment by malt3** [comment]
Regarding the signature verification: Ut only works when downloading the signature ([constellation-darwin-arm64.sig](https://github.com/edgelesssys/constellation/releases/download/v2.8.0/constellation-darwin-amd64.sig)) from the release page first.
There was a bug in our release pipeline that lead to signatures not being uploaded as release artifacts. I manually added the signatures for v2.8.0 to the release page now.
This will be fixed by #1917.
